indico/modules/rb/schemas.py

```diff
@@ -375,7 +376,7 @@ class CreateBookingSchema(mm.Schema):
     recurrence_weekdays = fields.List(fields.Str(validate=validate.OneOf(WEEKDAYS)))
     room_id = fields.Int(required=True)
     booked_for_user = Principal(data_key='user', allow_external_users=True)
-    booking_reason = fields.String(data_key='reason', validate=validate.Length(min=3), required=True)
+    booking_reason = fields.String(data_key='reason', validate=validate.Length(min=3), load_default='')
```

Comment:
Won't the min length validation fail when not specifying a reason?

Reply:
No, I did try to create a booking and it did it successfully.